catroid/src/main/java/org/catrobat/catroid/utils/CrashReporter.java

```diff
@@ -73,10 +69,6 @@ private static boolean isAnonymousReportEnabled() {
 		return preferences != null && preferences.getBoolean(SettingsActivity.SETTINGS_CRASH_REPORTS, false);
```

Comment:
I think we can't combine them. There can be some situation when isAnonymousReportEnabled() is True and isReportingEnabled() is False. 

For example: While the application is in standalone mode or debug mode then (isAnonymousReportEnabled is enabled from the debug.apk) but reporting is set to false. This is to prevent the reporting during the development mode. (if we will combine them then we will get emails from fabric for all the crashes from all the contributor of this application 😨).